helmfile/private-network-ibft-automated/besu-node/Chart.yaml

```diff
@@ -1,12 +1,14 @@
 apiVersion: v1
-name: pantheon-node
+name: besu-node
 version: 0.1.0
-description: Pantheon Nodes for a POA (IBFT2) Ethereum Blockchain
+description: Besu Nodes for a POA (IBFT2) Ethereum Blockchain
 appVersion: 1.1.4
 keywords:
   - validator
   - bootnode
   - node
+  - besu
+  - hyperledger
   - pantheon
   - ethereum
   - pantheon
```

Comment:
```suggestion

```

Reply:
I've removed `pantheon` - having pinged Arnaud to have a look at the helmfile as well